enarx-keep-sgx-shim/src/event.rs

```diff
@@ -18,70 +19,81 @@ pub extern "C" fn event(
     aex: &mut StateSaveArea,
     ctx: &Context,
 ) {
-    let mut h = Handler::new(layout, aex, ctx);
```

Comment:
You should leave this line here and not move the handler instantiation into the match structure. You create it in all cases anyway.

Reply:
@npmccallum I was getting some anger from the Rust borrow checker, but I'll try again.

Reply:
I got it.